schema.graphql

```diff
@@ -38,6 +38,9 @@ type Wallet @entity {
 
     "Next ID (transaction-id) for outgoing transactions - represents getTransactionCount(pending:true, executed:true)"
     nextId              : BigInt!
+
+    "Amount of ETH available for the Multisig wallet on GSN"
+    balanceGaugeGSN     : BigInt
```

Comment:
- Maybe use `balanceGSN` instead
- Type `BigInt!` for a mandatory field and initiate the field in `factory.ts`